### PR TITLE
simple destructuring tutorial

### PR DESCRIPTION
**Pull request by fgkolf**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/javascript/destructuring/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+const [firstFruit] = fruits;
+```
+Or just the second by ignoring the first one and so on:
+``` javascript
+const [, secondFruit] = fruits;  //ignores the first fruit
+const [,, thirdFruit] = fruits;  //ignores first and second fruit
+```
+
+Additionaly we can use the <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters' target='_blank' rel='nofollow'>`rest-operator`</a> to get some values and then pack the remaining in another variable:
+``` javascript
```
Comment (ezioda004) [inline guide/english/javascript/destructuring/index.md]: Can you change `rest-operator` to `rest-parameter`? In [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters#Destructuring_rest_parameters) and ES spec, its mentioned as rest parameter so we should stick to that.

File: guide/english/javascript/destructuring/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Object Destructuring
+The same technique can be used to get specific properties of an object in disctinct variables
+``` javascript
+const fruit = { name: 'apple', color: 'red', season: 'autumn' };
+const { name, color, season } = fruit; // name = 'apple', color = 'red', season = 'autumn'
+```
+
+The rest operator can be used to assing all remaining properties in a new object:
+``` javascript
```
Comment (ezioda004) [inline guide/english/javascript/destructuring/index.md]: Here too.

**Review by ezioda004 — CHANGES_REQUESTED** [review]
Great article, can you please make the small changes as requested? 
Thanks.

File: guide/english/javascript/destructuring/index.md (hunk 1 above)
Comment (fgkolf) [reply guide/english/javascript/destructuring/index.md]: done

File: guide/english/javascript/destructuring/index.md (hunk 2 above)
Comment (fgkolf) [reply guide/english/javascript/destructuring/index.md]: done

**Review by ezioda004 — APPROVED** [review]

**Comment by ezioda004** [comment]
Just noticed that theres an article in the `/es6` [folder](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/javascript/es6/destructuring/index.md).
Making this one duplicate - will discuss with others about *unpacking* the ES6 folder.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Hi @fgkolf could you check [this file](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/javascript/es6/destructuring/index.md) and make your changes over there

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**